src/blaze/middleware/guard.clj

```diff
@@ -0,0 +1,16 @@
+(ns blaze.middleware.guard
+  (:require
+   [buddy.auth :refer [authenticated?]]
+   [ring.util.response :as ring]))
+
+
+(defn wrap-guard
+  "If the request is unauthenticated return an  response."
+  [handler]
+  (fn [request]
+    (if (:tried-authentication request)
```

Comment:
I would remove the condition and make the whole guard middleware `identity` if we have no authentication enabled.